app/config/ConfigDecorator.scala

```diff
@@ -267,6 +267,10 @@ class ConfigDecorator @Inject() (
   lazy val sessionTimeoutInMinutes = sessionTimeoutInSeconds / 60
   lazy val sessionCountdownInSeconds = runModeConfiguration.getOptional[Int]("ptaSession.countdown").getOrElse(120)
 
+  lazy val incomeTaxViewChangeFrtonendHost = getExternalUrl(s"income-tax-view-change-frontend.host").getOrElse("")
+
+  lazy val itsaViewUrl = s"$incomeTaxViewChangeFrtonendHost/report-quarterly/income-and-expenses/view"
```

Comment:
The origin is missing on the itsa url:
/report-quarterly/income-and-expenses/view?origin=PTA

Reply:
Done